CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- `@Registrable.register(...)` decorator no longer masks the decorated class's annotations
```

Comment:
Could you put this under the `### Fixed` section below?

Reply:
Done.